ThirdParty/repo/org/nhind/xd-common/1.2-CONNECT/xd-common-1.2-CONNECT.pom

```diff
@@ -17,7 +17,7 @@
         </developer>   
         <developer>
             <name>Beau Grantham</name>
-            <email>bgrantha@harris.com</email>
+            <email>bgrantha@connect.com</email>
```

Comment:
Consider using one of the domains reserved for testing CONNECT's Direct polling, or example.org, instead of connect.com.